application/features/email.feature

```diff
@@ -213,6 +206,41 @@ Feature: Email
       | to send                 | does NOT exist | does exist      | delete_mfa        | should NOT  |
       | to send                 | does exist     | does exist      | delete_mfa        | should NOT  |
       | to send                 | does NOT exist | does NOT exist  | delete_mfa        | should      |
-      | Not to send             | does NOT exist | does NOT exist  | delete_mfa        | should NOT  |
+      | NOT to send             | does NOT exist | does NOT exist  | delete_mfa        | should NOT  |
       | to send                 | does NOT exist | does NOT exist  | create_mfa        | should NOT  |
-      | Not to send             | does NOT exist | does NOT exist  | create_mfa        | should NOT  |
\ No newline at end of file
+      | NOT to send             | does NOT exist | does NOT exist  | create_mfa        | should NOT  |
+
+  Scenario Outline: When to send refresh backup codes emails
+    Given we are configured <sendRefreshBackupCodesEml> refresh backup codes emails
+      And I remove records of any emails that have been sent
+      And no mfas exist
+      And a user already exists
+      And a backup code mfa option does exist
+      And there are <backupCodes> backup codes
+    When a backup code is used up by that user
+    Then a "refresh-backup-codes" email <shouldOrNot> have been sent to them
+      And a "refresh-backup-codes" email to that user <shouldOrNot> have been logged
+
+    Examples:
+      | sendRefreshBackupCodesEml | backupCodes    | shouldOrNot |
+      | to send                   | 9              | should NOT  |
+      | to send                   | 5              | should NOT  |
+      | to send                   | 4              | should      |
+      | to send                   | 1              | should      |
+      | NOT to send               | 5              | should NOT  |
+      | NOT to send               | 4              | should NOT  |
+
+  Scenario Outline: Check if an email type has been sent recently (negative days ago means not sent yet)
```

Comment:
Similarly here, should we add a line setting the `emailRepeatDelayDays` to something specific? That could protect us against the tests failing if we've set our local value for that to something other than the default.

Reply:
Done